examples/tahoe-base-template.pkr.hcl

```diff
@@ -0,0 +1,116 @@
+packer {
+  required_plugins {
+    macstadium-orka = {
+      version = "= 3.0.1"
```

Comment:
we need to update the version to the latest available.
Or change it here to say at least 3.0.1
I have seen customers using the examples directly and wondering why the plgin is not behaving correctly